support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -133,6 +133,23 @@ const legend = css`
 	}
 `;
 
+const personalDetailsFieldGroupStyles = (hideDetailsHeading?: boolean) => css`
+	position: relative;
+	margin-top: ${hideDetailsHeading ? `${space[4]}px` : '0'};
+
+	& > *:not(:first-of-type) {
+		margin-top: ${space[3]}px;
+	}
+	${from.tablet} {
+		& > *:not(:first-of-type) {
+			margin-top: ${space[4]}px;
+		}
+	}
+`;
+const personalDetailsHeader = css`
```

Comment:
Is this for screen readers?

Reply:
Yup - we should generally use [fieldsets](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) to describe form structure, and fieldsets need headings - in this case one we don't want to show visually.